package-lock.json

```diff
@@ -70,16 +86,38 @@
         "escape-html": "1.0.3"
       }
     },
+    "concat-map": {
```

Comment:
you have removed the babel from ```package.json``` file but where do the new dev dependencies come from?